addons/website/static/src/js/content/website_root.js

```diff
@@ -5,13 +5,14 @@ var core = require('web.core');
 var Dialog = require('web.Dialog');
 var publicRootData = require('web.public.root');
 require("web.zoomodoo");
+const KeyboardNavigationMixin = require('web.KeyboardNavigationMixin');
 
 var _t = core._t;
 
 var websiteRootRegistry = publicRootData.publicRootRegistry;
 
-var WebsiteRoot = publicRootData.PublicRoot.extend({
-    events: _.extend({}, publicRootData.PublicRoot.prototype.events || {}, {
+var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
+    events: _.extend({}, KeyboardNavigationMixin.events, publicRootData.PublicRoot.prototype.events || {}, {
```

Comment:
In fact I did not thing this through when telling you to use the website root... The website root is the top level widget for both visitors and connected users with edition access rights. We probably don't want the mixin to be used for visitors. Maybe you can use the navbar element or a new widget between the website root and the navbar element, I am not sure ?

Reply:
They keyboard navigation mixin adds shortcuts for all buttons on the page, I didn't see any reason not to allow keyboard navigation for visitors. After checking though, it looks like there are very few actual buttons visible to visitors so I think you're right. I think the navbar is a good fit, since fullscreen mode hides the navbar.

Reply:
In the end, as discussed in person, since navbar's element is not the body, it wouldn't catch keydown events, and since the navbar is loaded for visitors anyway, it wouldn't change anything in terms of loading, so we're leaving it in the website root for now. If at some point the navbar is split into its own assets for logged in users only, we can move this into the same asset as well.